### PR TITLE
Red styling

### PR DESCRIPTION
**Pull request by adamjohnsnow**

### CODE REVIEW

File: app/assets/stylesheets/footer.css [hunk 1]
```diff
@@ -15,14 +15,14 @@
  */
 
  .footer {
-   position: absolute;
+   position: fixed;
```
Comment (ffasolin) [inline app/assets/stylesheets/footer.css]: @adamjohnsnow could we maybe move this back to absolute? It's a bit annoying to have the footer fixed on screen when you're scrolling down - especially on a small screen.

**Review by ffasolin — CHANGES_REQUESTED** [review]
Would it be ok to change to

**Comment by adamjohnsnow** [comment]
Changes made as requested. Please approve.

**Review by ffasolin — APPROVED** [review]

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -29,17 +29,20 @@ body {
 }
 
 nav {
-  background-color: #445561;
-  height: 50px;
+  background-color: #964040;
+  height: 80px;
+  position: fixed;
+  z-index: 9999;/* set this to something high*/
```
Comment (neoeno) [inline app/assets/stylesheets/application.css.scss]: You call that a high number?!

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<div class="header-buffer"></div>
```
Comment (neoeno) [inline app/views/sessions/new.html.erb]: Imagine nagging every dev to add this to their views when they forget! Nightmare! Can it go somewhere in the layout? Near the nav?

**Review by neoeno — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.css.scss (hunk 2 above)
Comment (adamjohnsnow) [reply app/assets/stylesheets/application.css.scss]: z-index didn't seem to make a difference, removed.

File: app/views/sessions/new.html.erb (hunk 3 above)
Comment (adamjohnsnow) [reply app/views/sessions/new.html.erb]: Good point, moved.

**Review by neoeno — APPROVED** [review]
I wasn't even asking you to remove the `z-index`, but seems like you noticed it wasn't required anyway! Great!

**Pull request merged**